### ISSUE TITLE
pings displaying ultra-low on server list

### ISSUE THREAD

**Issue by RicksonFOLD** [FortressOne/server-qwprogs#1120]
![photo_2023-04-01_11-44-01](https://user-images.githubusercontent.com/127990133/229299516-0ad340c2-5479-4e25-94cb-2ab3248ad64f.jpg)

**Comment by drzel** [comment]
Yes those pings look wrong assuming you haven't relocated to Melbourne. Are you using a proxy? Not sure if that affects it.

**Comment by RicksonFOLD** [comment]
I don't believe I was using a proxy or had even connected to a server at the time I took the ss.

**Comment by RicksonFOLD** [comment]
One other suggestion I have is to sort the servers by server, then ping. Theoretically, I should ping the same to virginia pub as I do to virginia duel. Yet they are separated in the list by dallas pub. I would suggest sorting by ping of, say, the pub server, then port so all the cities' servers are grouped, then listed in order.

**Comment by haze-au** [comment]
Pings are a measurement at a point in times, it is an indicator and will vary in server, this is known.

Click the column header and it will sort by Hostname (this represents the Server Description and not the actual server address). 

Sorting by player numbers can be more useful if you actually want to play - the power is in your hands to sort!

P.S. Why use the server browser? Add some aliases to connect to servers in your config, or follow the links on FO discord :)
